src/cr/cube/cube.py

```diff
@@ -5,7 +5,12 @@
 CubeSet is the main API class for manipulating Crunch.io JSON cube responses.
 """
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
```

Comment:
This line was fine as it was. `isort` needs `-l 88` or `--line-length 88` to wrap at the same place as Black.